experimental/cloudstrapper/playbooks/ami-configure.yaml

```diff
@@ -2,7 +2,7 @@
 - hosts: "{{ aminode }}"
   roles:
     - magma-agw-1.5
-    - {role: clean-authorized-keys, tags: clearSSHKeys}
```

Comment:
I think there is a use-case where we do need to clean keys (for the base AMI generation). Can we still keep this one as well and have customers choose the option based on tags?

Reply:
This can be resolved. I see you have covered it in clean-ami below.